src/client/app/DragonflyPlayComponent.jsx

```diff
@@ -95,50 +95,51 @@ class DragonflyPlayComponent extends React.Component {
     var videoUrl = "https://s3-us-west-2.amazonaws.com/dragonfly-videos-transcoded/" + videoId + "/mp4-" + videoId + ".mp4";
 
     var thumbnailUrl = "./images/play.png";
+    var dragonfly = this.props.dragonfly;
+    if (dragonfly.logoId) {
+      var logo = "https://s3-us-west-2.amazonaws.com/dragonfly-logos/" + dragonfly.logoId;
+    }
```

Comment:
Dima, you use this logic and html template for a logo button multiple times. Better create separate single component to handle it all in one place